playwright-e2e/dsm/component/tables/onc-history-table.ts

```diff
@@ -179,10 +179,10 @@ export default class OncHistoryTable extends Table {
     }
 
     if (currentValue !== actualValue) {
+      await textarea.fill(actualValue, false);
       const [resp] = await Promise.all([
         waitForResponse(this.page, { uri: 'patch' }),
-        await textarea.fill(actualValue, false),
-        await textarea.blur()
+        textarea.blur(),
```

Comment:
`fill()` does not triggers `/patch` request but `blur()` does because it forces textarea to lose the focus.
Problem root cause is the actual order of completion of promises inside `Promise.all` is not guaranteed. `blur()` (sometimes) is completed before other promises.